src/main/kotlin/org/jitsi/nlj/Event.kt

```diff
@@ -16,11 +16,10 @@
 package org.jitsi.nlj
 
 import org.jitsi.utils.MediaType
-import org.jitsi_modified.impl.neomedia.rtp.MediaStreamTrackDesc
 
 interface Event
 
-class SetMediaStreamTracksEvent(val mediaStreamTrackDescs: Array<MediaStreamTrackDesc>) : Event
+class SetMediaSourcesEvent(val mediaSourceDescs: Array<MediaSourceDesc>) : Event
```

Comment:
Do we still need to use `Array` when dealing with these?

Reply:
I hadn't thought about it.  Do you think a `List` would be better?

Reply:
I remembered a lot of annoyances when dealing with the raw Arrays in the past, so I thought it would be nice...I don't know if there's anything about using ArrayList that would be problematic, though.